### PR TITLE
style footnotes business ethics

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #2824

- create mixin `modify_customNumberFootnotes` used to get the counter type set in `$Config_CustomNumberFootnotes` in the the config.scss file for books where a different counter type is needed 

- create a mixin `custom_numberFootnotes`  for custom numbering footnotes using counter type

### CODE REVIEW

**Comment by ajluk** [comment]
The link color will be added within this PR: https://github.com/openstax/cnx-recipes/pull/2823. I will update my branch then.

File: recipes/books/_generator.scss [hunk 1]
```diff
@@ -192,9 +192,13 @@
...
-    @include count_footnotes(footnotes);
-    @include count_footnoteLinks(footnoteLinks);
-    @include number_footnotes(footnotes, footnoteLinks);
+    @if $Config_CustomNumberFootnotes !=() {
+      @include modify_customNumberFootnotes($Config_CustomNumberFootnotes);
+    } @else {
+      @include count_footnotes(footnotes);
+      @include count_footnoteLinks(footnoteLinks);
+      @include number_footnotes(footnotes, footnoteLinks);
+    }
```
Comment (wilkus27) [inline recipes/books/_generator.scss:201]: I would recommend to enable `move-footnotes` flag for Business Ethics. Then this part of code shouldn't be changed.

File: recipes/mixins/_count.scss [hunk 2]
```diff
@@ -371,7 +371,7 @@
   @include _count(
     $name,
     'aside[role="doc-footnote"]',
-    '[data-type="chapter"], .appendix'
+    '[data-type="chapter"], .appendix',
```
Comment (wilkus27) [inline recipes/mixins/_count.scss:374]: Is this change necessary?

File: recipes/mixins/_number.scss [hunk 3]
```diff
@@ -647,6 +647,17 @@
...
+@mixin custom_numberFootnotes($footnoteCounter, $footnoteLinkCounter, $counterType) {
+  a[role="doc-noteref"] {
+    content: counter($footnoteLinkCounter, $counterType);
+  }
+  aside[role="doc-footnote"]::before {
+    data-type: "footnote-number";
+    content: counter($footnoteCounter, $counterType);
+  }
+}
+
```
Comment (wilkus27) [inline recipes/mixins/_number.scss:660]: I think we should have just one config for custom footnotes. It was already created for footnotes in precalc, but I just noticed that we didn't modify it after tags for footnotes changed, so it doesn't work now and footnotes in precalc are not numbered properly :disappointed: 

In this situation, I think we should keep all mixins created by you in this PR and I will create separate issue for footnotes in Precalc. Then some of us will modify your mixins to add additional option needed in this book and remove old mixins. What do you think?

**Review by wilkus27 — CHANGES_REQUESTED** [review]

File: recipes/mixins/_count.scss (hunk 2 above)
Comment (ajluk) [reply recipes/mixins/_count.scss]: No, I have just left it accidentally. Already removed! :)

File: recipes/mixins/_number.scss (hunk 3 above)
Comment (ajluk) [reply recipes/mixins/_number.scss:660]: Yes, it's a good idea. These changes could be rather time consuming.

**Review by wilkus27 — APPROVED** [review]

**Review by Stackblocks — APPROVED** [review]

**Pull request merged**